src/components/saunas/SaunaCard.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { Sauna } from '../../entities/Sauna'
+
+export type SaunaCardProps ={
+  sauna : Sauna,
+}
+
+const SaunaCard = (props:SaunaCardProps) => {
+  return (
+    <div className="grid gap-4">
+            <h2 className="text-2xl font-extrabold text-gray-900">{props.sauna.name}</h2>
+            <div>
+                <img 
+                    src={props.sauna.imgLink}
+                    alt={"props.alt"}
+                />
+            </div>
+            <div>
+                 <div>
+                   <h3 className="text-sm text-gray-700">
+                     <a href={"props.sauna.refLink"}>
```

Comment:
I would put a placeholder in code, rather than the string here. And you should use the <Link> of react.

Reply:
changed to <Link> tag and made necessary changes